src/Apps/Order2/Routes/Checkout/Components/Order2ReviewStep.tsx

```diff
@@ -91,28 +93,41 @@ export const Order2ReviewStep: React.FC<Order2ReviewStepProps> = ({
           </Text>
         </Box>
       </Flex>
-      <Box mb={2}>
+      <Box>
         <Order2PricingBreakdown order={orderData} />
       </Box>
+      <Spacer y={2} />
       <Message variant="default" p={1}>
         <Flex>
           <ShieldIcon fill="mono100" />
           <Spacer x={1} />
           <Text variant="xs" color="mono100">
-            Your purchase is protected with Artsy&rsquo;s Buyer Guarantee.
+            Your purchase is protected with{" "}
+            <RouterLink
+              onClick={() => checkoutTracking.clickedBuyerProtection()}
+              inline
+              target="_blank"
+              to={BUYER_GUARANTEE_URL}
+            >
+              Artsy&rsquo;s Buyer Guarantee
+            </RouterLink>
+            .
           </Text>
         </Flex>
       </Message>
-      <Spacer y={4} />
+      <Spacer y={2} />
       {stepState === CheckoutStepState.ACTIVE && (
-        <Button
-          variant={"primaryBlack"}
-          width="100%"
-          onClick={handleClick}
-          loading={loading}
-        >
-          Submit
-        </Button>
+        <>
+          <Spacer y={2} />
+          <Button
```

Comment:
Part of [EMI-2600] which I figured i should just do since adding the link just above was the other half of that ticket.